i18n/en.json

```diff
@@ -4383,6 +4383,14 @@
   "picture_selector.select_button.ariaLabel": "Select picture",
   "plan.cloud": "Cloud",
   "plan.self_serve": "Self-serve",
+  "pluggable_rhs.tourtip.boards.access": "Access your linked boards from the Boards icon on the right hand App bar.",
+  "pluggable_rhs.tourtip.boards.click": "Click into boards from this right panel.",
+  "pluggable_rhs.tourtip.boards.review": "Review board updates from your channels.",
+  "pluggable_rhs.tourtip.boards.title": "Access your {count} linked boards!",
+  "pluggable_rhs.tourtip.playbooks.access": "Access your linked playbooks from the Playbooks icon on the right hand App bar.",
+  "pluggable_rhs.tourtip.playbooks.click": "Click into playbooks from this right panel.",
+  "pluggable_rhs.tourtip.playbooks.review": "Review playbook updates from your channels.",
+  "pluggable_rhs.tourtip.playbooks.title": "Access your {count} linked playbook run.",
```

Comment:
Nit: pluralization between this and `"pluggable_rhs.tourtip.boards.title": "Access your {count} linked boards!",` is inconsistent. I don't care how we choose to do the pluralization, but we should probably be consistent since these will be seen close together.

Reply:
nice catch, thanks! done!